app/experimenter/experiments/api/v5/types.py

```diff
@@ -78,25 +76,27 @@ class NimbusLabelValueType(graphene.ObjectType):
 
 
 class NimbusCountryType(DjangoObjectType):
-    id = graphene.Int()
-    code = graphene.String()
-    name = graphene.String()
+    id = graphene.String(required=True)
+    code = graphene.String(required=True)
+    name = graphene.String(required=True)
 
     class Meta:
         model = Country
+        fields = ("id", "code", "name")
 
 
 class NimbusLocaleType(DjangoObjectType):
-    id = graphene.Int()
-    code = graphene.String()
-    name = graphene.String()
+    id = graphene.String(required=True)
+    code = graphene.String(required=True)
+    name = graphene.String(required=True)
 
     class Meta:
         model = Locale
+        fields = ("id", "code", "name")
 
 
 class NimbusProjectType(DjangoObjectType):
-    id = graphene.Int()
+    id = graphene.String()
```

Comment:
why we are changing from `int` to `String`????

Reply:
So.  Some fields in the frontend will read an int from the gql api, then pass it into a react component, like the multiselect, which will convert it into a string, and then it'll try to write the string back to the backend.  Ideally we'd convert it from a string back to an int in the client so the backend never knows about this, but because we're not using any strong typing in the frontend before we write back to the api, the type checking will pass but gql will throw a runtime error that it can't write a string to an int field.  

So until we get better type enforcement in the frontend, it's safer to make the backend both send and accept strings for fields that the frontend is converting so it's just a string the entire time, and since it's just a db id identifier, you can treat it as static data anyways so it doesn't really need to be an integer since we're not doing any integer validation on it, it's just a unique id.  The serializers will happily accept strings for integers and convert them back into ints before using them so this distinction only exists in the graphql layer.

So this is basically the safest way to deal with this now until we can do https://github.com/mozilla/experimenter/issues/7980